README.md

```diff
@@ -1,10 +1,10 @@
-# AMDCapstone
+# HPC Algorithm Performance Tracking Software
```

Comment:
Give it a nice creative name. This might be the hardest thing to ask for but have fun. You guys came up with Pargo for the parser to mongo, so I know you are capable of finding something creative.

Reply:
Yes, up to you. Leave it as this if you cannot come up with anything else.